app/Http/Controllers/ClassController.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\ClassData;
+
+
+class ClassController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        /* get all classes from db '
+        return view all classes, classes data
+        select * from classes = Class::get();*/
+        $classes = ClassData::get();
+        return view('classes', compact ('classes'));    
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     */
+    public function create()
+    {
+        return view('addclass');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(Request $request)
+    {
+    
+        #dd($request);
+        // fixed data
+        /*$title ='maths';
+        $capacity = '10';
+        $fulled = true;
+        $price ='190000';
+        $begTime = '14:05:19';
+        $endTime = '16:35:00';*/
+
+                // data from form
+
+
+
+/* elfk w tfasil str 'fulled' => isset($request->fulled):
+
+if(isset($request->fulled)){
+    $fulled = true;
+}else{
+    $fulled = false;    
+}
+l2n isset bs 4aila el values true w false keda keda*/
+$data = [
```

Comment:
I would like it if you care more about your code indentation, make sure to format your code after writing, note it is part of the project evaluation